### PR TITLE
fix graverobber bugs

### PR DESCRIPTION
**Pull request by Berthalamew**
- fixed suicide skull drops
- fixed issue where you had to pick up 1 extra skull to end the game

### CODE REVIEW

**Comment by Berthalamew** [comment]
- fixed bug with team games in graverobber

File: xlive/Blam/Engine/game/game_engine_util.cpp [hunk 1]
```diff
@@ -17,11 +17,11 @@ void game_engine_check_for_round_winner()
 	p_game_engine_check_for_round_winner();
 }
 
-void game_engine_end_round_with_winner(DWORD player_index, bool unk_bool)
+void game_engine_end_round_with_winner(DWORD player_index, bool go_to_next_round)
```
Comment (nukeulater) [inline xlive/Blam/Engine/game/game_engine_util.cpp]: in the end what is this first parameter, `player_index` or `player_team`?

Comment (Berthalamew) [reply xlive/Blam/Engine/game/game_engine_util.cpp]: it's actually both, i'll update the parameter name

Comment (Berthalamew) [reply xlive/Blam/Engine/game/game_engine_util.cpp]: changed to player_datum_or_team_index

**Pull request merged**